resources/hooks/useLogin.js

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { useNavigate, useParams } from "react-router";
+import { useUserStore } from "../js/useUserStore";
+import { apiUrl } from "../js/App";
+
+export const useLogin = () => {
+    const [message, setMessage] = useState("");
+    let navigate = useNavigate();
+    const { id } = useParams();
+    const { setUser } = useUserStore();
+
+    const login = async (email, password) => {
+        // make request first to sanctum/csrf-cookie endpoint
+        //to initialize CSRF protection for the application
+        axios.get("/sanctum/csrf-cookie").then(() => {
+            const payload = {
+                email,
+                password,
+            };
+            axios
+                .post(`${apiUrl}/${id}/user/login`, payload, {
```

Comment:
To remove company id in login url

Reply:
done!

Reply:
Plus transformed to onSubmit frpm onClick